src/pages/STG_SettingsManager.page

```diff
@@ -267,6 +267,16 @@
                                 <li class="slds-tree__item" role="treeitem" aria-level="2" id="idPanelErrorNotifNav"><a href="#" onclick="ShowPanel('idPanelErrorNotif');return false;" class="slds-grow slds-m-left_large">{!$Label.stgNavErrorNotify}</a></li>
                                 <li class="slds-tree__item" role="treeitem" aria-level="2" id="idPanelTDTMNav"><a href="#" onclick="ShowPanel('idPanelTDTM');return false;" class="slds-grow slds-m-left_large">{!$Label.stgNavTriggerConfig}</a></li>
                                 <li class="slds-tree__item" role="treeitem" aria-level="2" id="idPanelDataImportAdvancedMappingNav"><a href="#" onclick="ShowPanel('idPanelDataImportAdvancedMapping');return false;" class="slds-grow slds-m-left_large">{!$Label.bdiAdvancedMapping}</a></li>
+                                <li class="slds-tree__item"
+                                    role="treeitem"
+                                    aria-level="2"
+                                    id="idPanelGatewayManagementNav">
+
+                                    <a href="#"
+                                       onclick="ShowPanel('idPanelGatewayManagement');return false;"
+                                       class="slds-grow slds-m-left_large"> Elevate Gateway Management
+                                    </a>
+                                </li>
```

Comment:
Personally not a huge fan of the formatting on this page, but what do you think about adhering to the established one-line?

Reply:
@psadie-sfdo yeah I danced around what to do there. Didn't want to change the formatting of the rest of the page. I'll fix that to keep it consistent with the rest of the page. Just updated.